src/scripts/main.js

```diff
@@ -1,5 +1,12 @@
 'use strict';
 
 document.addEventListener('click', (e) => {
-  // write code here
+  const img = document.querySelector('.spider');
+
+  if (!e.target.closest('.wall')) {
+    return;
+  }
+
+  img.style.top = e.offsetY - img.clientHeight / 2 + 'px';
+  img.style.left = e.offsetX - img.clientWidth / 2 + 'px';
```

Comment:
Similar to the previous line, use `e.clientX` instead of `e.offsetX` to account for the page's scroll position.